### PR TITLE
Updated to include new KEYs

### PR DESCRIPTION
**Pull request by crnormand**

### CODE REVIEW

**Comment by crnormand** [comment]
Update to the output template HTML page that indicates the new KEYs available.

File: output_templates.html [hunk 1]
```diff
@@ -413,30 +382,58 @@ <h3>Spells Loop</h3>
...
+	  	  <li>MELEE_LOOP_END</li>
+	  	  <li>MELEE_LOOP_START</li>
+	  </ul>
+	  <h3>Hierarchical Melee Attacks Loop (one line for each weapon, and then one line for each mode of attack)</h3>
+	  <ul>
+	  	  <li>HIERARCHICAL_MELEE_LOOP_START</li>
+      	  <ul>
+      	  	  <li>ATTACK_MODES_LOOP_START</li>
+      	  	  <li>ATTACK_MODES_LOOP_END</li>
+      	  </ul>
```
Comment (richardwilkes) [inline output_templates.html]: This block should be indented appropriately

File: output_templates.html [hunk 2]
```diff
@@ -413,30 +382,58 @@ <h3>Spells Loop</h3>
...
+	<h3>Keys that can be used in any of the above Melee loops</h3>
+	  <ul>
           <li>BLOCK</li>
           <li>DAMAGE</li>
                     <li>DESCRIPTION<br>(Produces output of the form below, excluding those sections not present)<br>
                         <div class="code">@DESCRIPTION_PRIMARY&lt;div class="note"&gt;@DESCRIPTION_MODIFIER_NOTES&lt;/div&gt;&lt;div class="note"&gt;@DESCRIPTION_NOTES&lt;/div&gt;</div></li>
                     <li>DESCRIPTION_MODIFIER_NOTES<br>(Outputs the notes generated by active modifiers. Can be appended with <strong>_PAREN</strong> to surround the output in parenthesis)</li>
                      <li>DESCRIPTION_NOTES<br>(Outputs the text in the Notes field of the item. Can be appended with <strong>_PAREN</strong> to surround the output in parenthesis)</li>
                     <li>DESCRIPTION_PRIMARY<br>(Outputs the name of the item)</li>
+          <li>LEGAILITY_CLASS</li>
```
Comment (richardwilkes) [inline output_templates.html]: Mispelling... is it that way in the code, too? If so, it should be changed.

File: output_templates.html [hunk 3]
```diff
@@ -413,30 +382,58 @@ <h3>Spells Loop</h3>
...
+	  	  <li>RANGED_LOOP_END</li>
+	  	  <li>RANGED_LOOP_START</li>
+	  </ul>
+	  <h3>Hierarchical Ranged Attack Loop (one line for each weapon, and then one line for each mode of attack)</h3>
+	  <ul>
+	  	  <li>HIERARCHICAL_RANGED_LOOP_START</li>
+      	  <ul>
+      	  	  <li>ATTACK_MODES_LOOP_START</li>
+      	  	  <li>ATTACK_MODES_LOOP_END</li>
+      	  </ul>
```
Comment (richardwilkes) [inline output_templates.html]: Indent the block

**Comment by richardwilkes** [comment]
After taking a look at the final file, I noticed that the indentation in general was damaged. It looks like a mixture of spaces and tabs have been used. Please reformat using just spaces, with 2 spaces for indentation at each level.

**Comment by richardwilkes** [comment]
btw: I realize that the file already had some issues in the indentation area before you touched it -- but since you're in there, if you could normalize it, I'd appreciate it.

**Comment by crnormand** [comment]
Agreed.   It will look a lot better.

File: output_templates.html (hunk 2 above)
Comment (crnormand) [reply output_templates.html]: Ah, yes.   The literal was correct, but the variable name wasn't.   I will be fixed in my next 'gcs' pull request.

**Comment by crnormand** [comment]
Done.   I removed all tabs, and found a formatting tool on the internet and it seems to have done the trick.

**Comment by richardwilkes** [comment]
Unfortunately, that appears to have destroyed some of the other formatting, especially in the "code" blocks where pre-formatted text was. There are also some inconsistencies in the indentation, for example, the menu has seemingly arbitrary indentation towards its end for some reason.

**Comment by crnormand** [comment]
Oh fudge.   Ok, I will revert and hand edit.

**Comment by crnormand** [comment]
My apologies.    HTML is not my first language ;-), and I was relying on the tool to do the formatting.   I did not see what it had done to the code section.   It looks better now.

**Review by richardwilkes — DISMISSED** [review]

**Comment by richardwilkes** [comment]
Ugh. There are significant conflicts now because I forgot this was here and made other changes...

**Pull request closed without merging**